File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/RepresentativeComissionModel.ts
export interface RepresentativeComissionModel {
    comissionId: number | null;
    representativeId: number | null;
    representativeCode: string | null;

    comissionDate: Date | null;
    comissionValue: number | null;
    isApproved: boolean | null;
    comissionTypeId: number | null;
    notes: string|null;
}

File: SofiForce.Models/Models/EntityModels/RepresentativeComissionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class RepresentativeComissionModel
    {
		public Int32? ComissionId { get; set; }
		public Int32? RepresentativeId { get; set; }
		public DateTime? ComissionDate { get; set; }
		public decimal? ComissionValue { get; set; }
		public bool? IsApproved { get; set; }

		public Int32? ComissionTypeId { get; set; }
		public string Notes { get; set; }
	}
}
